editor/resources/meta.edn

```diff
@@ -155,6 +155,10 @@
    :label "Clear Color Alpha"
    :default 0,
    :path ["render" "clear_color_alpha"]}
+  {:type :integer,
+   :help "max number of collision objects, 128 by default",
+   :default 128,
+   :path ["physics" "max_count"]}
```

Comment:
Should it be `max_collision_object_count` ? I'm worried that `max_count` is a bit ambiguous. On the other hand, other max_count values are used to define the max count of components so maybe it is not so bad?

By the way, is there also a limit on the number of shapes?

Reply:
Fixed, not it is `max_collision_object_count `